artiq/firmware/runtime/rtio_clocking.rs

```diff
@@ -52,6 +99,7 @@ pub mod crg {
 
     #[cfg(not(has_rtio_clock_switch))]
     pub fn init() -> bool {
+        info!("Using internal RTIO clock");
```

Comment:
@Spaqin I think this message leads to the following confusing log messages when using `rtio_clock` with value `ext0_bypass_125`:

```
[2023-07-07 14:10:21] [     0.160319s]  INFO(runtime::rtio_clocking): using external RTIO clock with PLL bypass
[2023-07-07 14:10:21] [     0.285920s]  INFO(runtime::rtio_clocking::crg): Using internal RTIO clock
```

Reply:
Yep, we should remove that.

Reply:
Thanks. I agree, it's confusing and doesn't tell the whole story - it is indeed unnecessary - as currently on all platforms we use Si5324 anyway.